content/blog/suva-abredeversicherung/suva-abredeversicherung.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Suva Chatbot für Abredeversicherung
+permalink: /blog/suva-chatbot-fuer-abredeversicherung
+templateKey: blog-post
+image: ./suva-chatbot.png
+date: 2020-03-06T11:16:49
+author: Philip Schönholzer
+description: Seit ein paar Wochen läuft unser Chatbot bei der Suva, um die Besucher beim Abschluss der Abredeversicherung zu unterstützen.
+categories:
+  - Projekte
+---
+
+## Ziele
+
+Die Suva bietet eine temporäre Unfallversicherung für Private, welche für eine Weile nicht Arbeiten. Diese Versicherung nennt die Suva Abredeversicherung. Wie in der Versicherungsbranche leider üblich, ist man als Kunde nicht sicher, wie die Versicherung funktioniert, was die Voraussetzung für diese sind, und welche Leistungen gedeckt sind.
+
+Die Ziele des Chatbot-Projekts waren die folgenden:
+
+- höhere Conversionsrate bei Abredeversicherung
+- weniger Supportfälle
+- ein gutes Erlebnis für die Kunden
+
+Der Chatbots sollten die häufigsten Fragen der Besucher beantworten. Für den Fall, dass der Chatbot nicht mehr weiter weiss, sollte die Möglichkeit Live-Chats bestehen.
+
+## Umsetzung
+
+Zu Beginn des Projekts haben wir die Absichten, Antworten und Dialoge in zwei Workshops erarbeitet. Daraufhin implementierten wir die erarbeiteten Inhalte und stellten ein Test-Chatbot zur Verfügung. Dieser Chatbot konnte bereits zwei Wochen nach Projektstart getestet werden. Die Äusserungen welche der Chatbot nicht verstand, wiesen wir mittels Training den korrekten Absichten zu oder erfassten neue Absichten falls nötig. Zwischendurch gab es auch inhaltliche Diskussionen, wo wir im Team aber schnell gute Lösungen fanden.
```

Comment:
Zu Beginn des Projekts haben wir die Absichten, Antworten und Dialoge in zwei Workshops erarbeitet. Daraufhin implementierten wir die erarbeiteten Inhalte und stellten einen Test-Chatbot zur Verfügung. Dieser Chatbot konnte bereits zwei Wochen nach Projektstart getestet werden. Die Äusserungen, welche der Chatbot nicht verstand, wiesen wir mittels Training den korrekten Absichten zu oder erfassten neue Absichten, falls nötig. Zwischendurch gab es auch inhaltliche Diskussionen, wo wir im Team aber schnell gute Lösungen fanden.